.github/workflows/lint_and_format.yml

```diff
@@ -32,9 +34,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install pyink==24.3.0
+        pip install pytype
         pip install pylint
     - name: Check the code with pylint
       run: |
         pylint $(git ls-files '*.py')
-    - name: Check format with pyink
-      run: pyink --check .
+    - name: Run pytype
+      run: pytype --config=pytype-conf.cfg
```

Comment:
I'm wondering if we can keep both?

Reply:
my bad, will restore pyink, I tried to run both in precommit but failed